docs/sources/reference/components/otelcol/otelcol.receiver.syslog.md

```diff
@@ -0,0 +1,271 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.syslog/
```

Comment:
@clayton-cornell do we need to add a tag for public preview?

Reply:
Yes. I missed that  :-(  I saw the shared lookup... forgot to make sure the badge was added.

Parallel to this, we are looking at better ways of adding badges to the docs... so we can avoid HTML spans.